market/market/coingecko/coingecko_test.go

```diff
@@ -102,3 +104,58 @@ func Test_normalizeTickers(t *testing.T) {
 		})
 	}
 }
+
+func Test_normalizeTickersWithFilter(t *testing.T) {
+	prices := coingecko.CoinPrices{
+		{
+			Id:           "cUSDC",
+			Symbol:       "cUSDC",
+			CurrentPrice: 0.0021,
+			MarketCap:    minimalMarketCap + 1,
+		},
+		{
+			Id:           "cREP",
+			Symbol:       "cREP",
+			CurrentPrice: 0.02,
+			MarketCap:    minimalMarketCap,
+		},
+		{
+			Id:           "cREP",
+			Symbol:       "cREP",
+			CurrentPrice: 0.02,
+			MarketCap:    minimalMarketCap - 1,
```

Comment:
in the logic you use `>` aka `marketCap > minimalMarketCap`. Shouldn't the test fail? I'm not following

Reply:
Test assume that 3 objects will be normalized into 1 final object. 2 of them will be ignored